docs/sources/write/front-matter/index.md

```diff
@@ -96,23 +96,24 @@ To determine the relative alias, you must first understand the meaning of the cu
 
 For an alias in the page `/docs/grafana/latest/alerting/manage-notifications/`:
 
-- The current directory element (`.`) refers to the directory containing the current page.
+- The current directory element (`.`) refers to the directory containing the current page, _not_ the directory of the current page.
```

Comment:
```suggestion
- The current directory element (`.`) refers to the parent directory of the current page, _not_ the directory that contains the current page.  An easy way to remember is that one dot means you have moved one step back in the file path to the page.
```

Reply:
I think we should avoid "easy" if possible (https://developers.google.com/style/word-list#easy).

My only concern with this tip is that it is only true at the start of the alias, and that's really because the starting directory for any alias is the directory containing the current page.

I'm worried people will internalize `.` goes back one part and `..` goes back two when in actually `.` goes nowhere and `..` goes back one part.

Reply:
Ah, good point about "easy".  

Describing how aliases work is hard!